www/docs/en/dev/guide/platforms/ios/index.md

```diff
@@ -132,20 +203,20 @@ To preview the app in the iOS emulator:
 
 4. Press the __Run__ button that appears in the same toolbar to the
    left of the __Scheme__. That builds, deploys and runs the
-   application in the emulator. A separate emulator application opens
+   application in the simulator. A separate simulator application opens
    to display the app:
 
    ![]({{ site.baseurl }}/static/img/guide/platforms/ios/HelloWorldStandard.png)
 
-   Only one emulator may run at a time, so if you want to test the app
-   in a different emulator, you need to quit the emulator application
+   Only one simulator may run at a time, so if you want to test the app
+   in a different simulator, you need to quit the simulator application
    and run a different target within Xcode.
 
-Xcode comes bundled with emulators for the latest versions of iPhone
+Xcode comes bundled with simulators for the latest versions of iPhone
 and iPad. Older versions may be available from the __Xcode &rarr;
 Preferences &rarr; Downloads &rarr; Components__ panel.
 
-## Deploy to Device
+## Deploying to Device
```

Comment:
These sections should be moved under 'Project Configuration' above. We should not be speaking of deployment after having a section on 'Signing the app'.